.github/workflows/pr-create.yml

```diff
@@ -0,0 +1,41 @@
+name: TNP PR Created
```

Comment:
Let's rename this "Publish Preview" since this is what the workflow does. "pr created" is the event that triggers the workflow, but it is not the workflow itself.

Reply:
Updated